menas/src/main/scala/za/co/absa/enceladus/menas/services/SchemaRegistryService.scala

```diff
@@ -69,8 +120,34 @@ class SchemaRegistryService @Autowired()() {
    */
   def loadSchemaByUrl(remoteUrl: String): SchemaResponse = {
     try {
+      if (keyManagerFactory.isEmpty) {
```

Comment:
Can it work without `keyManagerFactory` or `trustManagerFactory`?
If yes, I would add a possibility to suppress the warnings.

Reply:
It can work with both being missing (not secured SR case). Theoretically, I can imagine only one being needed but makes little sense to me.

Reply:
Note from discussions: allow javax.ssl env properties be the default and specific config settings to override them?

Reply:
@benedeki - haha, having been thinking about this and having tested it, it actually works like this out-of-the-box. If one does not specify key/trust store, null is passed as the factory and in such a case, the "default security providers" are used 😉

Reply:
Great!